lib/index.js

```diff
@@ -1,7 +1,22 @@
 'use strict';
 
-const utils = require('./utils');
 const dom = require('./dom');
 const html = require('./html');
+const utils = require('./utils');
 const xml = require('./xml');
-Object.assign(module.exports, utils, dom, html, xml);
+
+const namespaces = {
+  dom,
+  html,
+  utils,
+  xml,
+};
+
+// Export all methods and namespaced interfaces
+Object.assign(module.exports,
+  namespaces,
+  dom,
+  html,
+  utils,
+  xml
+);
```

Comment:
The current babel configuration doesn't allow for `namespaces, ...namespaces` just yet